jquery.kinetic.js

```diff
@@ -226,6 +224,13 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if ($.isFunction(self.settings.selectStart)){
```

Comment:
So sorry, I've taken you in a circle! Tired checking of code. You were right first time. It did make sense, because we need to stop select start if we're using the target duh. Thanks